examples/twittermap/web/app/views/twittermap/index.scala.html

```diff
@@ -1,79 +1,102 @@
 
 @import controllers.TwitterMapApplication.DBType
 @(title: String, ws: String, startDate: String, endDate:Option[String], sentimentEnabled: Boolean, sentimentUDF: String, removeSearchBar: Boolean, predefinedKeywords: Seq[String], cacheThreshold: Option[String], querySliceMills: Option[String], isDrugMap: Boolean, sqlDB: DBType.Value) @main(title, ws, startDate, endDate, sentimentEnabled, sentimentUDF, removeSearchBar, predefinedKeywords, cacheThreshold, querySliceMills, isDrugMap, sqlDB){
-<div xmlns="http://www.w3.org/1999/html" ng-controller="AppCtrl">
+  <div xmlns="http://www.w3.org/1999/html" ng-controller="AppCtrl">
```

Comment:
General comment about this file: Don't change the format of the source code such as indention. Only change the lines that you modified. Otherwise, it is hard to track what your change is.